firefox-ios/firefox-ios-tests/Tests/XCUITests/TrackingProtectionTests.swift

```diff
@@ -209,10 +208,10 @@ class TrackingProtectionTests: BaseTestCase {
         navigator.goto(TrackingProtectionContextMenuDetails)
         // A page displaying the connection is secure
         waitForElementsToExist(
-            [
-                app.staticTexts["mozilla.org"],
-                app.staticTexts["Secure connection"]
-            ]
+                    [
+                        app.staticTexts["mozilla.org"],
+                        app.staticTexts["Secure connection"]
+                    ]
```

Comment:
Please do not change the indentation (unless it's swiftlint related).

Reply:
I don't understand this indentation. It's not very Swifty. lolol oh well